composer.json

```diff
@@ -1,8 +1,20 @@
 {
-    "require": {},
+    "require": {
+        "php": "^7.0"
+    },
     "autoload": {
         "psr-4": {
             "PHPOOP\\": "src"
         }
+    },
+    "config": {
+        "bin-dir": "bin"
```

Comment:
This is first time i see this config option. Very helpful IMO.

Reply:
First time seeing it too. Does it work any different than using just scripts? 
```
"scripts": {
        "phpcs": "vendor/bin/phpcs --standard=PSR12 ./src",
        "phpcbf": "vendor/bin/phpcbf --standard=PSR12 ./src",
        "test": "vendor/bin/phpunit ./tests"
    }
```
This is how i've done it, which does the same i believe? just without the need for the ruleset.xml and the bin-dir.

Reply:
Setting the bin dir is not required. As you mentioned the default is in vendor/bin and you can just use it from there. I always setting it to explicitly, just to show there are scripts that you should know about.

The ruleset is required because in this project we need to disable some rules. For example to allow multiple class declarations in one file.